test/models/response/emcsTfe/SubmitExplainDelayResponseSpec.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.response.emcsTfe
+
+import base.SpecBase
+import fixtures.SubmitExplainDelayFixtures
+import play.api.libs.json.{JsPath, JsSuccess, Json}
+
+class SubmitExplainDelayResponseSpec extends SpecBase with SubmitExplainDelayFixtures {
+
+  "SubmitExplainDelayResponse" - {
+
+    "should read from json" - {
+
+      "should read from json - ChRIS" in {
+        Json.fromJson[SubmitExplainDelayResponse](successResponseChRISJson) mustBe JsSuccess(successResponseChRIS, JsPath \ "receipt")
+      }
+
+    }
+
+    "should write to json" - {
+
+      "should read from json - EIS" in {
+        Json.fromJson[SubmitExplainDelayResponse](successResponseEISJson) mustBe JsSuccess(successResponseEIS, JsPath \ "message")
+      }
```

Comment:
This needs to be moved up into the 'reads' test section and the test renamed to: `for EIS response`

Reply:
corrected on https://github.com/hmrc/emcs-tfe-explain-delay-frontend/pull/32/commits/3ea9c2ff8170dd4d57e8d9ce66cf0ea178a02808